exporter/stackdriverexporter/stackdriver.go

```diff
@@ -132,3 +177,36 @@ func (sde *stackdriverExporter) ConsumeMetricsData(ctx context.Context, md data.
 
 	return nil
 }
+
+func agentForLibraryInfo(li *commonpb.LibraryInfo) string {
+	// The exporter must be the OpenCensus agent exporter because the spans
+	// have been written to the OpenCensus service.
+	return "opencensus-" + getLanguageName(li.Language) + " " + li.CoreLibraryVersion + "; ocagent-exporter " + li.ExporterVersion
+}
+
+func getLanguageName(l commonpb.LibraryInfo_Language) string {
```

Comment:
iirc proto can transform this in a string, any specific reason to have this function?

Reply:
Good point, I changed to calling `.String()` on the language enum. Initially I was hoping to make these agent labels match what is set from the libraries by default (e.g. `opencensus-node` rather than `opencensus-node_js`), but I think standardizing on the proto enum names is fine. I did lowercase it so that it's a little more similar to the way that agent labels get set on the per-language exporters.